### PR TITLE
Milestone 3 : Add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by dqarias**
In this milestone we:

- Create forms with HTML5 validations.
- Add the contact form in the footer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # Hello Microverse
```
Comment (gtekle) [inline README.md]: - [ Optional ] You should update the project title from `Hello Microverse` to `Portfolio` to reflect the project you are working on.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
 
-> This is the second milestone of my portafolio project. For this part I set up it using best practices (linters, Github Flow). I built the middle section of the mobile website following the requirements that Microverse provide us.
+> This is the third milestone of my portafolio project. For this part I and My coding partnert  have worked using pair programing methodology. We set up the project using best practices (linters, Github Flow). We built the footer section of the mobile website following the requirements that Microverse provide us.
```
Comment (gtekle) [inline README.md]: - [ Optional ] The description section you README is for the overall project - Portfolio in this case. Therefore, you should add a description for the whole project. But, you can add to it as you progress through the module.

File: index.html [hunk 3]
```diff
@@ -1,172 +1,311 @@
...
+            <a href="#">Skills<img src="./images/IconRight.svg" alt="" /></a>
+          </li>
+        </ul>
+      </section>
     </main>
-</body>
-</html>
\ No newline at end of file
+    <footer class="footer">
+      <fieldset>
```
Comment (gtekle) [inline index.html]: - [ ] The semantic use of `<fieldset>` is to group related form elements together. Therefore, you should remove it. Please see this [reference](https://www.w3schools.com/tags/tag_fieldset.asp)

File: index.html [hunk 4]
```diff
@@ -1,172 +1,311 @@
...
+        <form
```
Comment (gtekle) [inline index.html]: - [ ] The semantically meaningful ordering of elements in the footer section, can be something like:
```
<footer>
    <section class="contact-me">
        <div>
            <h2> Contact Me </>
            <p> If you have an application you... </p>
        <div>
        <form action= >
        </form>
    </section>
    
    // here you can add other section such as foot navigation links, etc later on.
</footer>
```

File: index.html [hunk 5]
```diff
@@ -1,172 +1,311 @@
...
+          method="POST"
+        >
+          <h3 class="form-title">Contact me</h3>
+          <p class="form-description">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d
+            love to help with it
+          </p>
+          <ul class="form-list">
+            <li>
```
Comment (gtekle) [inline index.html]: - [ ] In my opinion, you can make your form more leaner and easy to read by removing the use of `<ul>` and its `<li>` from here. You can use something like:
```
<form>
   <input name />
   <input email />
   <textarea message />
   <button type="submit" />
</form>
```

**Review by gtekle — CHANGES_REQUESTED** [review]
Hi **@TEAM** :wave:, 

Good job so far! :clap: :clap: :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!
dog-pug



## Highlights
:heavy_check_mark: Nice UI
:heavy_check_mark: Meaningful commit messages
:heavy_check_mark: Good use of GitHub flow

## Required Changes ♻️
_Check the comments under the review._

- [ ] You should use the padding of 24px for the form elements in the contact-me section as in the Figma design. Please take a look at the snapshot for a device size 375px width:

#### Figma Design    
contact-form-padding-figma 
### Implementation
contact-form-padding-impl-375px |

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 6]
```diff
@@ -1,12 +1,12 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Hello Microverse
+# My Portafolio Project
```
Comment (koomekevo) [inline README.md:3]: @dqarias Kindly rectify this typo as it should be `Portfolio` instead of `Portafolio`

**Review by koomekevo — CHANGES_REQUESTED** [review]
Hi @dqarias, 



Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — COMMENTED** [review]
Hello @dqarias 👋,

_This is an additional review as a result of the second opinion request._

- I can see the only requested change by the last reviewer was an OPTIONAL suggestion by the first CR as a result the project should have been approved 🙏 

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approved right away :)

Keep rocking 🚀!

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @dqarias   , 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well implemented Figma design.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**